eox_nelp/signals/tests/test_receivers.py

```diff
@@ -14,7 +14,7 @@
 from ddt import data, ddt
 from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.test import override_settings
+from django.test import TestCase, override_settings
```

Comment:
Why you need to use Django test case?

Reply:
Becasue unittest.TestCase doesn't work with reverse relations, so things like user.extrainfo fails